GPU/GX2/VertexShaderGeneratorGX2.cpp

```diff
@@ -22,15 +22,23 @@
 #include "GPU/GX2/VertexShaderGeneratorGX2.h"
 #include "GPU/Vulkan/VertexShaderGeneratorVulkan.h"
 #include "GPU/GX2/GX2StaticShaders.h"
+#include "GPU/ge_constants.h"
 
 #include <wiiu/os/debug.h>
 
 void GenerateVertexShaderGX2(const VShaderID &id, GX2VertexShader *vs) {
 	// TODO;
 	*vs = STVshaderGX2;
+
+	if (id.Bit(VS_BIT_IS_THROUGH) && id.Bit(VS_BIT_HAS_COLOR)) {
+		*vs = clearVShaderGX2;
+	} else if (id.Bit(VS_BIT_HAS_COLOR) && id.Bit(VS_BIT_DO_TEXTURE)) {
+		*vs = cTexVShaderGX2;
```

Comment:
Hmm.  This seems like a hard road in the end for all graphics and to match other backends.  Does it seem crazy to build the ops here (e.g. using a pair of vectors)?

-[Unknown]

Reply:
that's the plan! 
the shaders are the most challenging part here so I'm keeping those last.
once I can render cube.elf correctly with the hardcoded shaders, I'll remove them and move to dynamic generation instead.
I might need to add a pre-compiled ubershader as fallback, until I can support all bits in the shader ID.